assets/test/Swift-Markdown/Package.swift

```diff
@@ -0,0 +1,23 @@
+// swift-tools-version:6.0
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    // FIXME: Can be changed back to Swift-Markdown when
+    // https://github.com/swiftlang/swift-package-manager/issues/7931
+    // is released in the toolchain
+    name: "swift-markdown",
```

Comment:
is this arbitrary or must it be "swift-markdown" because we want it to overshadow the remote dependency? If the latter, should have a comment "needs to be swift-markdown for .... test to verify ...."

Reply:
You are right this is not arbitrary, I will add comment to make this more obvious.